modules/contentbox/modules/contentbox-ui/interceptors/CBRequest.cfc

```diff
@@ -107,17 +107,24 @@ component extends="coldbox.system.Interceptor" {
 			}
 		}
 
+		var linkLogout = "#prc.cbAdminEntryPoint#/security/doLogout";
+
 		// Render the admin bar out
 		var adminBar = renderView(
 			view   = "adminbar/index",
 			module = "contentbox-ui",
 			args   = {
 				oContent       : oContent ?: javacast( "null", "" ),
 				linkEdit       : linkEdit,
+				linkLogout     : linkLogout,
 				oCurrentAuthor : prc.oCurrentAuthor
 			}
 		);
-		cfhtmlhead( text = "#adminBar#" );
+		var adminBarCSS = renderView( view = "adminbar/adminbarCSS", module = "contentbox-ui" );
+		buffer.append( "#adminBar#" );
+
+		// Append adminbar styles to the html head
+		cfhtmlhead( text = "#adminBarCSS#" );
```

Comment:
add the `renderView()` inline. We dont' need any variable assignments

```
cfhtmlhead( text = renderView( .... ) )
```

Reply:
Done.